_posts/2024-04-12-a-time-machine-for-your-data-opensearch-searchable-snapshots.md

```diff
@@ -0,0 +1,147 @@
+
+We have decided to retain data for 90 days on disk and to move all our older data to S3 storage, whereas previously, our team had to spend hours performing a full restore from older snapshots. Using searchable snapshots, that data can be queried immediately. Our index policy was configured as follows:
+
+```json
+{
+  "policy": {
+    "description": "Rollover indexes every day, delete indexes after 90 days.",
+    "error_notification": null,
+    "default_state": "hot",
+    "states": [
+      {
+        "name": "hot",
+        "actions": [],
+        "transitions": [
+          {
+            "state_name": "rolled-over",
+            "conditions": {
+              "min_index_age": "1d"
+            }
+          }
+        ]
+      },
+      {
+        "name": "rolled-over",
+        "actions": [
+          {
+            "retry": {
+              "count": 3,
+              "backoff": "exponential",
+              "delay": "1m"
+            },
+            "rollover": {}
+          }
+        ],
+        "transitions": [
+          {
+            "state_name": "deleting",
+            "conditions": {
+              "min_index_age": "91d"
+            }
+          }
+        ]
+      },
+      {
+        "name": "deleting",
+        "actions": [
+          {
+            "retry": {
+              "count": 3,
+              "backoff": "exponential",
+              "delay": "1m"
+            },
+            "delete": {}
+          }
+        ],
+        "transitions": []
+      }
+    ],
+    "ism_template": [
+      {
+        "index_patterns": [
+          "audit-logging-*"
+        ],
+        "priority": 150
+      }
+    ]
+  }
+}
+```
+
+By spending significantly less time searching through old logs, our security team can spend more time on more valuable activities. This means we’re saving on resources across the business.
+
+### Bringing the past into the present
+
+We hope you can see the numerous benefits of using OpenSearch searchable snapshots. They can save you time by making remotely stored snapshots instantly accessible. They can also save you effort by removing the need to restore an entire snapshot back to the cluster. Lastly, they can save you money by reducing the amount of data being stored to disk in the cluster.
+
+But perhaps the biggest benefit of searchable snapshots is that they give your historical data a portal back into the present day, making it suddenly accessible again. Like a time machine, they bring your data to you from the past, instantly making it more useful and valuable. If you’re looking for the ability to easily and instantly travel into the past to find the data you need, searchable snapshots is the ideal solution.
+
+If you would like to find out more about searchable snapshots, read our release blog post or contact our friendly support team at support@instaclustr.com.
```

Comment:
Add link to your r`elease blog post`: https://www.instaclustr.com/blog/searchable-snapshots-now-available-for-opensearch-on-the-instaclustr-managed-platform/

Reply:
Thanks for catching that! We have updated the link to the `release blog post` and applied a `mailto` link to `support@instaclustr.com`